pkgs/by-name/cl/clock-tui/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  installShellFiles,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "clock-tui";
+  version = "0.5.0";
+  src = fetchFromGitHub {
+    owner = "race604";
+    repo = "clock-tui";
+    tag = "v${version}";
+    hash = "sha256-sXfGm36WgL6h06cT9TowkaEaBB1roe4qP3OW2JJ3EwQ=";
+  };
+
+  cargoHash = "sha256-oiQ8RiCPP2PCQv+oUb299ijWZHL1xR13/UPvVeMojYQ=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  preFixup = ''
+    $out/bin/xtask
+    rm -f $out/bin/xtask
+
+    installManPage assets/gen/tclock.1
+
+    installShellCompletion --cmd tclock \
+      --bash <(cat assets/gen/tclock.bash) \
+      --fish <(cat assets/gen/tclock.fish) \
+      --zsh <(cat assets/gen/_tclock)
+  '';
+
+  meta = {
+    description = "A clock app in terminal written in Rust, supports local clock, timer and stopwatch.";
```

Comment:
The description should not start with ˋAˋ and should not end with ˋ.ˋ
See https://nixos.org/manual/nixpkgs/stable/#var-meta-description